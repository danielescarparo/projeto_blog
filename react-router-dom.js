import React from "react";

export const Link = () => <a> link </a>;
export const NavLink = ({children}) => <div> {children} </div>;