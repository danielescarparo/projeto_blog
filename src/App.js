import React, { Component } from 'react';
import {NavLink as Link} from 'react-router-dom'

import './App.css';

class App extends Component {
  render() {
    return (
      <div>
        <div className="container">        
          <h1 className="titulo"> Bloguinho</h1>
          <div className="opcoes">
            <Link to="/opcao1">opcao1</Link>
            <Link to="/opcao2">opcao2</Link>
          </div>
          <div>
        </div>
      </div>
      </div>  
    );
  }
}

export default App;
