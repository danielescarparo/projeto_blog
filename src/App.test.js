import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import Adapter from 'enzyme-adapter-react-16'; 
import { shallow, configure } from 'enzyme'; 
import {BrowserRouter, Switch, Router} from 'react-router-dom'
configure({adapter: new Adapter()});

it('renders without crashing', () => {
  const wrapper = shallow(<App />);
  expect(wrapper.html()).toMatchSnapshot();
});

