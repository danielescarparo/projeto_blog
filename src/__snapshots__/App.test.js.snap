// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `"<div><div class=\\"container\\"><h1 class=\\"titulo\\"> Bloguinho</h1><div class=\\"opcoes\\"><div> opcao1 </div><div> opcao2 </div></div><div></div></div></div>"`;
